rpcd/etc/openstack_deploy/user_osa_variables_defaults.yml

```diff
@@ -72,6 +72,10 @@ keystone_token_driver: "sql"
 # Galera overrides
 galera_cluster_name: rpc_galera_cluster
 
+# Ensure that xtrabackup is installed from the percona repo. The version that is hardcoded
+# in OSA M & N is too old for current holland, so we need to use the repo version.
+use_percona_upstream: yes
```

Comment:
Until artifacts, I'd use something like this instead:

percona_arch_url:
  x86_64: "https://www.percona.com/downloads/XtraBackup/Percona-XtraBackup-2.3.6/binary/debian/xenial/x86_64/percona-xtrabackup_2.3.6-1.xenial_amd64.deb"
percona_arch_sha256:
  x86_64: "16a16ee40e6095cdfb5d0547818fa8313d130debdd6f579fa453c97f0775a0fc"

or for 2.4 (maybe that's a move too big?):
percona_arch_url:
  x86_64: "https://www.percona.com/downloads/XtraBackup/Percona-XtraBackup-2.4.5/binary/debian/xenial/x86_64/percona-xtrabackup-24_2.4.5-1.xenial_amd64.deb"
percona_arch_sha256:
  x86_64: "6a71704e625194689d2a6b325765608040ff5da82ac4bc8ce6ac3756f9a9130f"

works on N branch of OSA, but requires a -e override (which is ok with our user_osa_variables_defaults).